src/main/assets/scss/main.scss

```diff
@@ -1,10 +1,34 @@
 @import '~govuk-frontend';
 
+$line-colour: #1d70b8;
+
 .govuk-header__navigation--end {
   margin: -5vh 0 0;
   padding: 1vh 0 0;
 }
 
+.govuk-section-break--visible {
+  border-color: $line-colour;
+}
+
+.table-header {
+  border-spacing: 100px;
+}
+
+.govuk-table {
```

Comment:
do we need this here? are we not using standard gov uk table layout?

Reply:
I've removed the the wrapping / spacing, however left the table-layout in.

The fixed width layout makes sure the columns are equal in size

Reply:
ah ok, have you seen `govuk-!-width-two-thirds` and the like? if so then all good, but they also have spacing classes, under their `width` folder

Reply:
Ah, yeah I have indeed. I did actually play around with those originally, however there isn't one unfortunately for one-sixth, which is the one we would need for our table.

Reply:
👍🏼  nice job checking it all out :)